apps/web/components/Organization/Wrapper/Themes/VinnueftirlitidTheme/VinnueftirlitidHeader.tsx

```diff
@@ -20,6 +21,13 @@ const VinnueftilitidHeader: React.FC<React.PropsWithChildren<HeaderProps>> = ({
     [organizationPage.organization?.namespace?.fields],
   )
   const n = useNamespace(namespace)
+  const { activeLocale } = useI18n()
+  const logoAltText = n(
```

Comment:
Just in case there isn't a title for the organization perhaps it'd be better to fallback to an empty string instead of potentially concatenating undefined or '' with ' Frontpage'